elements/pfe-styles/src/pfe-typography-mobile.scss

```diff
@@ -0,0 +1,6 @@
+/* Quick demo for shrinking everything on mobile; may not be desirable */
```

Comment:
Is this still a demo or is it part of the typography package?  Do we need to update this comment?

Reply:
I moved the CSS into the index.html demo file so we *can* remove this, what's your preference?

Reply:
I think remove if it's not being used